scripts/test_downgrade_from_tag.sh

```diff
@@ -10,7 +10,7 @@ TEST_VERSION=${TEST_VERSION:-v2}
 TEST_TMPDIR=${TEST_TMPDIR:-$(mktemp -d 2>/dev/null || mktemp -d -t 'timescaledb_downgrade_test' || mkdir -p /tmp/${RANDOM})}
 UPDATE_PG_PORT=${UPDATE_PG_PORT:-6432}
 CLEAN_PG_PORT=${CLEAN_PG_PORT:-6433}
-PG_VERSION=${PG_VERSION:-12.0}
+PG_VERSION=${PG_VERSION:-14.0}
```

Comment:
wasn't there an issue for timescale with PG 14.0 and this was fixed in 14.1?

Reply:
I think what you're talking about is the explain output for Memoize nodes changed in 14.2, right?

Reply:
Make sense bump it to 14.2 as we already for the other github actions. Any complains @svenklemm ??

Reply:
fine with me